__tests__/components/ActiveLink.test.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
```

Comment:
I came across this eslint annoyance in my AMO port as well. I added the following rule to `.eslintrc`:

```
    "jsx-a11y/anchor-is-valid": [
      "error",
      {
        "components": ["Link"],
        "specialLink": ["hrefLeft", "hrefRight"],
        "aspects": ["invalidHref", "preferButton"]
      }
    ]
```